test/hand/format-as-lines.test.ts

```diff
@@ -0,0 +1,17 @@
+import { Hand, StringParser } from '../../src';
+describe('Testing converting hand to list of strings', () => {
```

Comment:
Can you add the exact name of the function you're testing, ie `Hand.formatHandAsLines`. Just makes it clear which is being tested.